python/sdssdb/__init__.py

```diff
@@ -32,6 +33,8 @@
 config = get_config(NAME)
 
 autoconnect = True
+use_psycopg3 = os.environ.get("SDSSDB_PSYCOPG3", "false")
```

Comment:
We may want to document this somewhere, what this flag is, and what it's needed for.

Reply:
They should. I'll add something to the docs.